lambdas/code_exec/Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM public.ecr.aws/lambda/nodejs:12
+
+
+FROM public.ecr.aws/bitnami/golang:latest
```

Comment:
I think this should be one specifically for lambdas and it should be the base image. Then put a generic Node one on top.

Reply:
changing in separate pr